test/SimpleToken.js

```diff
@@ -101,377 +102,377 @@ const SimpleToken = artifacts.require("./SimpleToken/SimpleToken.sol")
 //
 
 contract('SimpleToken', (accounts) => {
-   
-   const SYMBOL         = "ST"
-   const NAME           = "Simple Token"
-   const DECIMALS       = 18
-   const TOTAL_SUPPLY   = new BigNumber(web3.toWei(800000000, "ether"));
-      
-   const ST10000 = new BigNumber(web3.toWei(10000, "ether"));
-   const ST9000 = new BigNumber(web3.toWei(9000, "ether"));
-   const ST1000 = new BigNumber(web3.toWei(1000, "ether"));
-   const ST10 = new BigNumber(web3.toWei(10, "ether"));
-   const ST1 = new BigNumber(web3.toWei(1, "ether"));
-
-   const owner  = accounts[0]
-   const admin  = accounts[1]
-   const ops    = accounts[2]
-
-   async function createToken() {
-      return await SimpleToken.new()
-   }
-
-
-   describe('Basic properties', async () => {
-
-      var token = null
-
-      before(async () => {
-         token = await createToken()
-      })
-
-
-      it("name", async () => {
-         assert.equal(await token.name.call(), NAME)
-      })
-
-      it("symbol", async () => {
-         assert.equal(await token.symbol.call(), SYMBOL)
-      })
-
-      it("decimals", async () => {
-         assert.equal(await token.decimals.call(), DECIMALS)
-      })
-
-      it("totalSupply", async () => {
-         assert.equal((await token.totalSupply.call()).toNumber(), TOTAL_SUPPLY.toNumber())
-      })
 
-      it("balances is private", async () => {
-         assert.isTrue(typeof(token.balances) == 'undefined')
-      })
+  const SYMBOL = "ST"
```

Comment:
@benjaminbollen  @jasonklein @abhayks1 
SimpleToken.js was at 4 space indentation, however, we are following 2-2-2 space indentation. Due to this, the diff is looking larger than expected. 
I hope this is fine with you.